fs/expose/dokan/__init__.py

```diff
@@ -89,22 +100,46 @@
 import logging
 logger = logging.getLogger("fs.expose.dokan")
 
+if six.PY3:
```

Comment:
Same here, I think it should be the other way round, i.e. check for PY2 as the special case and consider Python3 the normal case. Should Python 2 ever disappear, this also makes it easier to remove this code. Ideally one could also use the `six.moves` module.